docs/src/main/asciidoc/logging.adoc

```diff
@@ -241,13 +241,34 @@ All potential properties are listed in the <<loggingConfigurationReference,loggi
 
 === Logging categories
 
-Logging is done on a per-category basis, with each category being configured independently.
-A category configuration recursively applies to all subcategories of that category unless there is a more specific matching sub-category configuration.
+Logging is configured on a per-category basis, with each category being configured independently.
+Configuration for a category applies recursively to all subcategories unless there is a more specific subcategory configuration.
 
-The parent of all logging categories is called the "root category".
-This category, being the ultimate parent, may contain configuration which applies globally to all other categories. This includes the globally configured handlers and formatters.
+The parent of all logging categories is called the "root category."
+As the ultimate parent, this category might contain a configuration that applies globally to all other categories.
+This includes the globally configured handlers and formatters.
+
+.An example of a global configuration that applies for all categories:
```

Comment:
Specificity of example: Should this be:
```suggestion
.An example of a global configuration that applies to all subcategories:
```

Reply:
Thanks for the hint with `to`. I think that both works, but will remain the `categories`.

Reply:
@MichalMaler  - Just one comment ^^. I don't see the updates in the current preview to verify. Otherwise all looks good :+1: